docs/development/priority-classes.md

```diff
@@ -19,14 +19,14 @@ When using the `gardener-operator` for managing the garden runtime and virtual c
 
 ### `PriorityClass`es for Garden Control Plane Components
 
-| Name                              | Priority  | Associated Components (Examples)                                                          |
-|---------------------------------- |-----------|-------------------------------------------------------------------------------------------|
-| `gardener-garden-system-critical` | 999999550 | `gardener-operator`, `gardener-resource-manager`                                          |
-| `gardener-garden-system-500`      | 999999500 | `virtual-garden-etcd-events`, `virtual-garden-etcd-main`, `virtual-garden-kube-apiserver` |
-| `gardener-garden-system-400`      | 999999400 |                                                                                           |
-| `gardener-garden-system-300`      | 999999300 | `vpa-admission-controller`, `etcd-druid`                                                  |
-| `gardener-garden-system-200`      | 999999200 | `vpa-recommender`, `vpa-updater`, `hvpa-controller`                                       |
-| `gardener-garden-system-100`      | 999999100 |                                                                                           |
+| Name                              | Priority  | Associated Components (Examples)                                                                    |
+|---------------------------------- |-----------|-----------------------------------------------------------------------------------------------------|
+| `gardener-garden-system-critical` | 999999550 | `gardener-operator`, `gardener-resource-manager`                                                    |
+| `gardener-garden-system-500`      | 999999500 | `virtual-garden-etcd-events`, `virtual-garden-etcd-main`, `virtual-garden-kube-apiserver`, `istio`  |
```

Comment:
In seeds, istio does have the same priority like gardenlet/GRM (ref https://github.com/gardener/gardener/blob/878377d9cb41214634fd01b48c035da190304eba/docs/development/priority-classes.md#priorityclasses-for-seed-system-components). Perhaps we should do the same in garden clusters, i.e. assign `gardener-garden-system-critical`?